WordPress/Classes/ViewRelated/System/WPSplitViewController.swift

```diff
@@ -370,6 +370,8 @@ class WPSplitViewController: UISplitViewController {
             } else {
                 navigationController.scrollContentToTopAnimated(animated)
             }
+
+            navigationController.setNavigationBarHidden(false, animated: animated)
```

Comment:
Hmm... would it be better to move this into the if clause so its only performed when `popToRootViewController` is called?